src/workerd/io/trace-stream.c++

```diff
@@ -360,6 +358,10 @@ jsg::JsValue ToJs(jsg::Lock& js, const tracing::Onset& onset, StringCache& cache
   auto obj = js.obj();
   obj.set(js, TYPE_STR, cache.get(js, ONSET_STR));
   obj.set(js, EXECUTIONMODEL_STR, cache.get(js, enumToStr(onset.workerInfo.executionModel)));
+  // TODO
```

Comment:
Todo

Reply:
My mistake, the need for overwriting this was already resolved, so I put the proper code path back in.